tests/test_close_help_post.py

```diff
@@ -0,0 +1,73 @@
+import datetime as dt
+from types import SimpleNamespace
+from typing import Any
+
+import pytest
+from githubkit.exception import RequestFailed
+
+from tests.utils import kitposer as kp
+
+from app.components.close_help_post import mention_entity
+from app.components.entity_mentions.discussions import DISCUSSION_QUERY
+from app.components.entity_mentions.models import GitHubUser
+
+
+def fill_entity(**kwargs: Any) -> SimpleNamespace:
+    defaults: dict[str, Any] = {
+        "number": 0,
+        "title": "",
+        "body": None,
+        "html_url": "https://github.com/foo/bar/issues/1",
```

Comment:
Self promo is very yum 😋.
```suggestion
        "html_url": "https://github.com/trag1c/zig-codeblocks/issues/1",
```
Also I didn't expect it but it coincidentally happens to be about testing too 💀.